### DISCUSSION TITLE
Обустройство пространства

### DISCUSSION THREAD

**Discussion by tort32** [hackerembassy/space#1] [Ideas]
Обсуждение по мебели и организации рабочего пространства.

В спейсе вроде бы достаточное количество столов и появились табуретки, рассадить теперь можно всех.
Много всякого стаффа у нас распихано по коробочкам, но но не хватает места для организации хранения. Все эти ящики валяются сейчас на полу, на столах, под столами, а должно на быть полках и стиллаже.

От @Assargadon еще было предложение приобрести систему хранения для радиодеталей. Удобнее в виде шкафчика-касетницы в выдвижными ящичками.

**Comment by Assargadon** [comment]
А может ещё лучше (и, вероятно, проще) начать с личных коробок и стеллажей под них. Это устоявшаяся хакспейсовская практика, насколько я понимаю. Почти что де-факто стандарт.

Смысл в том, что пластиковые прозрачные коробки купить гораздо проще и дешевле, чем хорошие кассетницы под радиодетали. И пока оборудованного места под них нету, эти ящики могут стоять друг на дружке. И, в отличие от радиодеталей, личный шкафчик создаёт более сильное...ощущение ассоциированности с местом.

Конечно, там возникают политические вопросы:
1. как связано наличие шкафчика и уплата пожертвований/членских взносов?
2. кто получает право на шкафчики, а кто - нет?

**Comment by Linka1386** [comment]
Пераработать музыкальную комнату.

1. Нужно высвободить табуретки, на которых сейчас лежит столешка (брус есть)
2. Сделать подсветку с выключателем (rgb-лента?)
3. Повесить полку для ноута/найти телик/монитор.
4. Повесить колонки. (Удлинить провода, если понадобится)
5. Купить диско шар для антуражности 😂 💃🕺(опционально)

Концепт согласовать с @cake64, @KagiChan, @arcturuss, @sunpunsch и другими гитаристами 😊

**Reply by tort32** [reply]
Про подсветку - https://github.com/hackerembassy/space/issues/7

**Reply by tort32** [reply]
Колонки возможно бы подошли как раз те, что сейчас на первом этаже.
Но к ним нужен сабвуфер, если кто-то найдет б/у-шный, у нас есть усилитель под него.
